request-an-access-token.md

```diff
@@ -45,6 +45,7 @@ permission scope changes. The PR should describe the permission scopes requested
 Repo                              | Secret name                   | Expiration date | Pull Request                               |
 ---                               | ---                           | ---             | ---                                        |
 [`nodejs/import-in-the-middle`][] | `RELEASE_PLEASE_GITHUB_TOKEN` | 2025-07-23      | <https://github.com/nodejs/admin/pull/902> |
```

Comment:
fwiw I renamed this secret to `RELEASE_PLEASE_GITHUB_TOKEN - iitm` to make it recognizable. I didn't know about this registry.

Reply:
It would be worth documenting what string to use on the bot's settings side (e.g. `<name of the repo>: <Secret name>`)

Reply:
yeah, there are others like https://github.com/nodejs/remark-preset-lint-node that probably need a similar token, but probably shouldn't be the same one